plutus-chain-index/app/Marconi.hs

```diff
@@ -49,16 +55,12 @@ import Streaming.Prelude qualified as S
 --     > select slotNo, datumHash, datum from kv_datumhsh_datum where slotNo = 39920450;
 --     39920450|679a55b523ff8d61942b2583b76e5d49498468164802ef1ebe513c685d6fb5c2|X(002f9787436835852ea78d3c45fc3d436b324184
 
-data IndexerType = DatumIndexer
-                 | UtxoIndexer
-                 deriving (Show, Eq)
-
 data Options = Options
-  { optionsSocketPath   :: String,
-    optionsNetworkId    :: NetworkId,
-    optionsChainPoint   :: ChainPoint,
-    optionsDatabasePath :: FilePath,
-    optionsIndexerType  :: IndexerType
+  { optionsSocketPath :: String,
+    optionsNetworkId  :: NetworkId,
+    optionsChainPoint :: ChainPoint,
+    optionsUtxoPath   :: Maybe FilePath,
+    optionsDatumPath  :: Maybe FilePath
```

Comment:
Curious, is it possible to put the data from the indexers in the same database?

Reply:
In the case of SQLite, most likely not due to higher contention.